Microsoft.Toolkit.Uwp.Notifications/Adaptive/BaseTextHelper.cs

```diff
@@ -8,12 +8,12 @@ namespace Microsoft.Toolkit.Uwp.Notifications
 {
     internal class BaseTextHelper
     {
-        internal static Element_AdaptiveText CreateBaseElement(IBaseText curr)
+        internal static Element_AdaptiveText CreateBaseElement(IBaseText baseText)
```

Comment:
I was going to comment here that we could leave this as `current` still for consistency with the change made in `BaseImageHelper.cs`

Reply:
@michael-hawker I was going to make this change for you.... but it looks like PR was merged already.  Tell me if you'd like this modified back to "current".